docs/contribute/general/reporting_issues.rst

```diff
@@ -0,0 +1,14 @@
+Reporting Issues
+================
+
+Contributions in the form of issue reports are highly appreciated as they help to improve
+the quality of pyfar packages and projects.
+
+Here are a couple of things to consider:
+
+- We provide templates for issue reports on `GitHub <https://github.com/pyfar/>`_ to help you provide all necessary information. When opening a new issue, please select the most appropriate template and make sure to fill out all required fields.
+- When reporting bugs, please make sure to include a minimal reproducible example as well as information about your environment (e.g., operating system, Python version, pyfar version, and any other relevant packages).
+- You may also help categorizing issues by applying appropriate labels.
+- If you are unsure about how to report an issue or which template to use, please feel free to reach out to us through `Slack <https://join.slack.com/t/pyfar/shared_invite/zt-2eacdhww2-iUiPnh_wuqg2zD939wL4kw>`_.
```

Comment:
I would suggest to also mention writing an email here, since this is less effort than registering for the slack channel.

Reply:
Good point.